Sources/Swift/Protocol/Codable/SentryLogAttributeCodable.swift

```diff
@@ -0,0 +1,42 @@
+extension SentryLog.Attribute: Codable {
+    
+    private enum CodingKeys: String, CodingKey {
```

Comment:
`m`: I think we could directly add the codable implementation to the Attribute enum. We only declared all the other codable implementations as extensions because we added them to existing objc classes. The same applies for `extension SentryLog: Codable` and